devel/tree-info.lua

```diff
@@ -0,0 +1,163 @@
+--Print a tree_info visualization of the tree at the cursor.
+--@module = true
+
+local branch_dir =
+{
+    [0] = ' ',
+    [1] = string.char(26), --W
+    [2] = string.char(25), --N
+    [3] = string.char(217), --WN
+    [4] = string.char(27), --E
+    [5] = string.char(196), --WE
+    [6] = string.char(192), --NE
+    [7] = string.char(193), --WNE
+    [8] = string.char(24), --S
+    [9] = string.char(191), --WS
+    [10] = string.char(179), --NS
+    [11] = string.char(180), --WNS
+    [12] = string.char(218), --ES
+    [13] = string.char(194), --WES
+    [14] = string.char(195), --NES
+    [15] = string.char(197), --WNES
+}
+
```

Comment:
in case the script has an error while accessing the data, it would keep the console in a better state if the print statements were self-contained in their color changes. That is, have a function like:

```lua
local function print_color(color, s)
    dfhack.color(color)
    dfhack.print(s)
    dfhack.color(COLOR_RESET)
end
```

and use that instead of `dfhack.print` directly